api/manifest/manifest.go

```diff
@@ -567,6 +567,16 @@ func (m *Manifest) Run(app string, cache bool, shift int) []error {
 		sch := make(chan error)
 		go (*m)[name].runAsync(m, m.prefixForEntry(name, i), app, name, cache, shift, ch, sch)
 		<-sch // block until started successfully
+
+		// block until we can get the container IP
+		for {
+			host := containerName(app, name)
+			ip, err := Execer("docker", "inspect", "-f", "{{ .NetworkSettings.IPAddress }}", host).CombinedOutput()
+
+			if (err == nil) && (net.ParseIP(strings.TrimSpace(string(ip))) != nil) {
+				break
+			}
```

Comment:
Can also probably be Output() instead of CombinedOutput() since you're only interested in Stdout